srcpkgs/wireproxy/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'wireproxy'
+pkgname=wireproxy
+version=1.0.1
+revision=0
+build_style=go
+go_import_path="github.com/octeep/wireproxy"
+go_package="${go_import_path}/cmd/wireproxy"
+short_desc="Wireguard client that exposes itself as a socks5 proxy"
+maintainer="Wind Wong <octeep@pm.me>"
+license="ISC"
+homepage="https://github.com/octeep/wireproxy"
+distfiles="https://github.com/octeep/wireproxy/archive/v${version}.tar.gz"
+checksum=e21eac22ef1b12dc2d7b3e5b58dcff183af9c1547be3e55a7e6c71394f053e1e
+
+system_accounts="wireproxy"
+caddy_homedir="/var/lib/wireproxy"
+
+conf_files="/etc/wireproxy/config"
+
+make_dirs="
+	/etc/wireproxy 0700 wireproxy wireproxy
```

Comment:
this should be owned by root, i think. also it only needs to be made if it's empty, which it is not if you put the config in there

Reply:
I explicitly created the folder to be owned by _wireproxy for reasons stated above.